.travis.yml

```diff
@@ -20,4 +20,4 @@ install:
 script:
   - source activate snakemake
   # run the workflow
-  - snakemake --use-conda --directory .test || (for f in .test/logs/*; do echo $f; cat $f; done; for f in .test/logs/*/*; do echo $f; cat $f; done; exit 1)
+  - snakemake --cores all --use-conda --directory .test || (for f in .test/logs/*; do echo $f; cat $f; done; for f in .test/logs/*/*; do echo $f; cat $f; done; exit 1)
```

Comment:
BTW, I think this `.travis.yml` file is an artifact from a time, when CI tests were run via Travis CI. But we have long moved to GitHub Actions, so all testing harness definitions are found in `.github/workflows/`. I guess we can simply remove this file here in the near future.